### DISCUSSION TITLE
Submitting maps for ranked anonymously.

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#30257] [Ideas]
Exactly what the title says.

I don't know, after watching the pishifat video from a few years ago where he went 'undercover', this just seems like a compelling idea. I'm not a good mapper by any means, I personally would never use this feature, but wouldn't it be really cool?

A few reasons why this could be awesome, imagine someone makes the equivalent to Impulse by handsome, some god-tier map, it gets mods, gets qualified and eventually ranked. It could be the pinnacle of modern mapping, and if nobody snitches, it will stay a secret who the mapper was.

Another reason, someone makes a map that mimics a super prominent mapper's style, like rrtyui or Hollow Wings, the map gets ranked and everyone's just under the impression that it's their map. Some time passes, and eventually rrtyui or Hollow Wings comes out on twitter and announces.. It wasn't their map? whaaaaaa???

Now that I'm done gawking about how cool this idea is, I'm going to list some issues/questions/downsides(?):

- **How does this get sorted?** An idea I have would be to upload all the beatmaps to one account ID. If a map from the "Anonymous" account gets ranked, then the person who submitted the map **would not** receive credit. 

- **Wouldn't this be a pain for people who run modding/BN queues?** Yes. On the forums, you would have to account for _anonymous posts_, it would be an option that you'd have to toggle before making a thread, it'd be weird, yes. Also, now that people run modding queues on third-party sites, there's going to be **at least** one person who has 40-50 pending mod requests from _Anonymous_.

- **How do you account for people abusing the system?** Even though I proposed one account to store all the Anonymous maps, there still has to be a way to identify who uploaded the map that can't be identified by anyone outside of staff members. Once that is implemented properly, then the owner of a modding queue can report an Anonymous user, and if the behavior is found to be abusive, then they would be revealed and proper discipline would be enforced.

I hope y'all like my idea. It was thought of on a whim, but it could have a lot of potential!

**Comment by bdach** [comment]
If there's one thing that I've learned about the internet over however many years I've interacted with it, it's that most people absolutely cannot be trusted with being fully anonymous. Maybe I'm cynical but I can't see this causing anything other than a bunch of flaming and baiting and other abhorrent behaviour that will add extra strain on the NAT and GMT in needing to clean that sewage out after it blows out.

**Comment by totallywaffles** [comment]
It would already be possible for someone to set up an anonymous modding/nomination queue with exported map files if someone wished to do that, and past that I don't see any functional purpose for this idea other than it being 'cool'.
